app/src/main/java/com/example/bigowlapp/viewModel/ViewUserViewModel.java

```diff
@@ -0,0 +1,24 @@
+package com.example.bigowlapp.viewModel;
```

Comment:
This class is unused, so I assume it should be removed.

Reply:
It has been removed

Reply:
I think its still in the repo.

Reply:
ok removed